examples/protocols/displayport/displayport-src.stanza

```diff
@@ -0,0 +1,143 @@
+#use-added-syntax(jitx)
+defpackage jsl/examples/protocols/displayport/displayport-src :
+  import core
+  import jitx
+  import jitx/commands
+
+  import jsl/symbols/SymbolDefn
+  import jsl/symbols/box-symbol
+  import jsl/landpatterns/packages
+  import jsl/landpatterns/BGA
+
+  import jsl/design/settings
+  import jsl/si/helpers
+  import jsl/pin-assignment
+  import jsl/bundles
+  import jsl/protocols/displayport
+
+doc: \<DOC>
+@brief DisplayPort dummy component
+The component defined in this file is an example of how to enable a DisplayPort interface
+on a component. The component demonstrates the use of JSL land-patterns and symbols.
+<DOC>
+
+val body = PackageBody(
+  width = 15.0 +/- 0.1,
+  length = 15.0 +/- 0.1,
+  height = 0.71 +/- [0.06, 0.0]
+)
+
+val BGA-pkg = BGA(
+  num-leads = 49,
+  rows = 8,
+  columns = 8,
+  lead-diam = 0.5,
+  pitch = 2.0,
+  package-body = body,
+  density-level = DensityLevelB
+)
+
+doc: \<DOC>
+@brief DisplayPort dummy component definition
+This definition assumes a BGA component with pins defined per the DisplayPort standard. The component has a single 
+DP source or sink port per the argument.
+@param source:True|False defines whether the component is a DP source (true) or DP sink (false)
+<DOC>
+
+public pcb-component component :
+  reference-prefix = "U"
+  mpn = "JITX002"
+  description = "Dummy device with both a source and sink DP port"
+
+  pin-properties :
+    [pin:Ref    | pads:Ref ... ]
+
+    [DP-TX-P[0] | A[1] ]
+    [DP-TX-N[0] | A[2] ]
+    [DP-TX-P[1] | D[4] ]
+    [DP-TX-N[1] | D[3] ]
+    [DP-TX-P[2] | D[2] ]
+    [DP-TX-N[2] | D[1] ]
+    [DP-TX-P[3] | C[7] ]
+    [DP-TX-N[3] | C[6] ]
+
+    [DP-RX-P[0] | B[3] ]
+    [DP-RX-N[0] | B[2] ]
+    [DP-RX-P[1] | A[8] ]
+    [DP-RX-N[1] | A[7] ]
+    [DP-RX-P[2] | C[4] ]
+    [DP-RX-N[2] | C[3] ]
+    [DP-RX-P[3] | C[2] ]
+    [DP-RX-N[3] | C[1] ]
+
+    [AUX-N      | C[5] ]
+    [AUX-P      | D[5] ]
+    [HPD        | D[6] ]
+
+    [VDD        | E[1] ]
+    [GND        | E[2] ]
+
+  val bus-r = Ref("bus")
+  val ctl-r = Ref("aux")
+  val pwr-r = Ref("pwr")
```

Comment:
I learned a neat trick yesterday. It isn't documented so probably shouldn't include it yet. 

`#R(bus)` will map to `Ref("bus")`. 

Provides a short hand for creating refs - especially indexed and field refs.